src/Explorer/ContextMenuButtonFactory.tsx

```diff
@@ -140,6 +140,21 @@ export const createCollectionContextMenuButton = (
   return items;
 };
 
+export const createSampleCollectionContextMenuButton = (
+  selectedCollection: ViewModels.CollectionBase
+): TreeNodeMenuItem[] => {
+  const items: TreeNodeMenuItem[] = [];
+  if (userContext.apiType === "SQL") {
+    items.push({
+      iconSrc: AddSqlQueryIcon,
+      onClick: () => selectedCollection && selectedCollection.onNewQueryClick(selectedCollection, undefined),
```

Comment:
The new sql query button should open the query copilot tab.

Reply:
Fixed :)